src/OpenFeature.Contrib.Providers.Statsig/EvaluationContextExtensions.cs

```diff
@@ -0,0 +1,70 @@
+﻿using OpenFeature.Model;
+using Statsig;
+
+namespace OpenFeature.Contrib.Providers.Statsig
+{
+    internal static class EvaluationContextExtensions
+    {
+        //These keys match the keys of the statsiguser object as descibed here
+        //https://docs.statsig.com/client/concepts/user
+        internal const string CONTEXT_APP_VERSION = "appVersion";
+        internal const string CONTEXT_COUNTRY = "country";
+        internal const string CONTEXT_EMAIL = "email";
+        internal const string CONTEXT_IP = "ip";
+        internal const string CONTEXT_LOCALE = "locale";
+        internal const string CONTEXT_USER_AGENT = "userAgent";
+        internal const string CONTEXT_PRIVATE_ATTRIBUTES = "privateAttributes";
+
+        public static StatsigUser AsStatsigUser(this EvaluationContext evaluationContext)
+        {
+            if (evaluationContext == null)
+                return null;
+
+            var user = new StatsigUser();
+            foreach (var item in evaluationContext)
+            {
+                //TODO: Await release containing this https://github.com/open-feature/dotnet-sdk/pull/231 to use TargetingKey instead of UserId
```

Comment:
I'm checking on this to see if we can release it soon.

Reply:
Thank you, if it's close we might as well wait for that.

Reply:
We released 1.5 last night.

Reply:
Thanks, @beeme1mr. Would you be ok with me doing the update [here](https://github.com/open-feature/dotnet-sdk-contrib/blob/533dfa652011b645d48c5d17dbdfe39b64a8eadf/build/Common.props#L27) as part of this PR, or does it need to be in a separate one?

Reply:
Sure, that would be fine with me.

Reply:
In the end I've decided to include it in this PR as it was a minor update. Please let me know if you would like me to change that so it goes in its own PR.